specs/protocol/jovian/exec-engine.md

```diff
@@ -128,9 +139,12 @@ It takes on a default value as described in the section on [L1 Attributes](./l1-
 
 ### Rationale
 
-While the current L1 fee mechanism charges for DA usage based on an estimate of the DA footprint of a transaction, it
-does not influence future base fee calculations. As a result, excessive DA usage is not efficiently reflected in the fee
-market, leading to suboptimal resource prices.
-
-By changing the meaning of the `gasUsed` field in times of high DA demand, the fee market can properly adjust without reverting
-to priority fee auctions (an inferior experience for users).
+While the current L1 fee mechanism charges for DA usage based on an estimate of the DA footprint of a transaction, no
+protocol mechanism currently reflects the limited _DA throughput_. E.g. on Ethereum with Pectra enabled, the available
+blob throughput is `~96 kB/s` (with a target of `~64 kB/s`), but the calldata floor gas price of `40` for calldata-heavy
+transactions allows for more incompressible transaction data throughput than even that of most single OP Stack chains.
+This is currently mitigated at the policy level by batcher-sequencer throttling. However, since batcher throttling
+artificially constricts block building, this can lead to actually falling base fees, unnecessary losses for chain
```

Comment:
```suggestion
This is currently mitigated at the policy level by batcher-sequencer throttling: a mechanism which
artificially constricts block building. This can cause base fees to fall, which implies unnecessary losses for chain
```

I prefer something like this since the specs don't otherwise mention throttling, so it needs a little introduction.